src/components/Checkbox.tsx

```diff
@@ -1,8 +1,8 @@
-import React from 'react';
-import { withStyles, createStyles, Theme } from '@material-ui/core/styles';
-import { TextField, Checkbox } from '@material-ui/core';
+import { Checkbox, TextField } from '@material-ui/core';
```

Comment:
Can you also add a `TODO` to rename this component? I was confused why this checkbox had a text field also. I don't know a good name off the top of my head though. Will think.